src/styles/bloks/stars.scss

```diff
@@ -0,0 +1,32 @@
+  .stars {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    margin: 17px 0 22px;
+
+    &-wrap {
+      display: flex;
+      margin-left: 16px;
+    }
+
+    &-wrap__img {
+      margin: 0 4px 0 0;
+      height: 16px;
+      width: 16px;
+      background: url(/src/images/star.svg) no-repeat;
+      background-position: center;
+
+      &:nth-child(-n+4) {
```

Comment:
Consider using styles from the previous task. The modifier should work correctly